test/funnels/automation/StableDepositorUniV3.t.sol

```diff
@@ -128,63 +141,66 @@ contract StableDepositorUniV3Test is DssTest {
             uint32 zzz,
             uint96 amt0,
             uint96 amt1,
-            uint96 req0,
-            uint96 req1,
+            uint96 req0_,
+            uint96 req1_,
             uint32 hop
         ) = stableDepositor.configs(address(0x123), address(0x456), uint24(314), 5, 6);
         assertEq(num,  23);
         assertEq(zzz,  0);
         assertEq(amt0, uint96(7));
         assertEq(amt1, uint96(8));
-        assertEq(req0, uint96(9));
-        assertEq(req1, uint96(10));
+        assertEq(req0_, uint96(9));
+        assertEq(req1_, uint96(10));
         assertEq(hop,  3600);
     }
 
     function testDepositWithdrawByKeeper() public {
         uint256 prevDai = GemLike(DAI).balanceOf(address(buffer));
         uint256 prevUsdc = GemLike(USDC).balanceOf(address(buffer));
-        (int32 initNum,,,,,,) = stableDepositor.configs(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100);
+        (int32 initNum,,,,,,) = stableDepositor.configs(DAI, USDC, uint24(100), tickLower, tickUpper);
         assertEq(initNum, 10);
         uint32 initialTime = uint32(block.timestamp);
 
-        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), REF_TICK-100, REF_TICK+100, uint128(491 * WAD), uint128(491 * 10**6));
+        vm.prank(KEEPER); stableDepositor.deposit(DAI, USDC, uint24(100), tickLower, tickUpper, req0, req1);
```

Comment:
Here before we used min amounts larger than req0 and req1 (there is different logic in case it is smaller/equal/larger).
Are we changing that on purpose?

Reply:
I think the code path taken is actually the same whether we pass `amt0Min = req0` or `amt0Min = req0 + 1`.

Reply:
I mean if the code in the stable depositor would not allow a number larger than `req` for some reason (since there is a bug) then `req0 + 1` would fail, but `req0` would not.